### PR TITLE
Feature/task 27820 01

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**

### CODE REVIEW

File: agenda-webapps/src/main/webapp/vue-app/agenda/components/event/AgendaEventDialog.vue [hunk 1]
```diff
@@ -123,7 +143,9 @@ export default {
       this.event = agendaEvent;
     },
     close() {
+      const defaultUrl = `${eXo.env.portal.context}/${eXo.env.portal.portalName}/agenda`;
       this.dialog = false;
+      window.history.replaceState('', '', defaultUrl);
```
Comment (boubaker) [inline agenda-webapps/src/main/webapp/vue-app/agenda/components/event/AgendaEventDialog.vue]:
```suggestion
      window.location.replace(window.location.pathname);
```

Comment (hassenelkamel-exo) [reply agenda-webapps/src/main/webapp/vue-app/agenda/components/event/AgendaEventDialog.vue]: done but the page will reload

File: agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaEventServiceImpl.java [hunk 2]
```diff
@@ -228,6 +233,7 @@ public Event createEvent(Event event,
     reminderService.saveEventReminders(createdEvent, reminders, userIdentityId);
     attendeeService.saveEventAttendees(createdEvent, attendees, userIdentityId, sendInvitation, false);
 
+    Utils.broadcastEvent(listenerService, "exo.agenda.event.added", eventId, 0);
```
Comment (boubaker) [inline agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaEventServiceImpl.java]: You didn't used `sendInvitation` flag. In fact, this listener shouldn't be triggered for notifications. May be for analytics only.

Comment (hassenelkamel-exo) [reply agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaEventServiceImpl.java]: = i will change it with Utils.broadcastEvent(listenerService, "exo.agenda.event.added", eventId, sendInvitation);

**Pull request closed without merging**